big-practice/src/style/components/table.css

```diff
@@ -0,0 +1,38 @@
+table tr td {
+  padding-top: 16px;
+  padding-bottom: 16px;
+}
+
+.table-wrap {
+  overflow: auto;
+}
+
+.table {
+  min-width: 700px;
+}
+
+.table-header {
+  background: var(--primary-color-50);
+  color: var(--primary-color-300);
+}
+
+.table-item-header {
+  font-size: var(--font-size-s);
+  line-height: var(--line-height-s);
+  font-weight: var(--font-weight-l);
+  align-items: center;
+  letter-spacing: 0.003em;
+  width: 5%;
```

Comment:
By default, we should not style for width because you don't know how many columns you have